### PR TITLE
Eperez eppn actions

### PR DESCRIPTION
**Pull request by enriquepablo**
eppn in secret box for token logins

eppn as user identifier in actions

### CODE REVIEW

File: src/eduid_webapp/oidc_proofing/app.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def init_oidc_proofing_app(name, config):
 
     app = eduid_init_app(name, config)
     app.config.update(config)
+    app.config['CELERY_CONFIG']['MONGO_URI'] = app.config['MONGO_URI']
```
Comment (johanlundberg) [inline src/eduid_webapp/oidc_proofing/app.py]: How come this is necessary now?

Comment (enriquepablo) [reply src/eduid_webapp/oidc_proofing/app.py]: The tests were failing and I added that as a temporary measure meaning to investigate the issue further later on. I forgot :-/

Comment (enriquepablo) [reply src/eduid_webapp/oidc_proofing/app.py]: I think I understand the problem now. It stems from the fact that we (necessarily) install all dependencies for all apps when testing webapp. Also in the docker images, if I'm not mistaken. The eduid-action package has a setuptools entry point for the AM, in which it tries to register a plugin to fetch attributes -in this case, the ToUs. And the initialization of this plugin needs to establish a mongo connection. I'll paste below the traceback I'm seeing.

A possible solution might be to move the am plugin code out of the eduid-action package, and provide it in a -amp package to only be installed in the AM app.

Another solution of course is to provide MONGO_URI in the celery config for our webapps ;-)

Comment (enriquepablo) [reply src/eduid_webapp/oidc_proofing/app.py]: ERROR: test_authenticate (eduid_webapp.oidc_proofing.tests.test_app.OidcProofingTests)                                                                                  
----------------------------------------------------------------------                                                                                                  
Traceback (most recent call last):                                                                                                                                      
  File "/home/eperez/src/git/eduid/eduid-webapp/src/eduid_webapp/oidc_proofing/tests/test_app.py", line 83, in setUp                                                    
    super(OidcProofingTests, self).setUp()                                                                                                                              
  File "/home/eperez/src/git/eduid/eduid-common/src/eduid_common/api/testing.py", line 98, in setUp                                                                     
    self.app = self.load_app(config)
  File "/home/eperez/src/git/eduid/eduid-webapp/src/eduid_webapp/oidc_proofing/tests/test_app.py", line 92, in load_app                                        [69/8389]
    return init_oidc_proofing_app('testing', config)                                                                                                                    
  File "/home/eperez/src/git/eduid/eduid-webapp/src/eduid_webapp/oidc_proofing/app.py", line 50, in init_oidc_proofing_app                                              
    app = mail_relay.init_relay(app)                                                                                                                                    
  File "/home/eperez/src/git/eduid/eduid-common/src/eduid_common/api/mail_relay.py", line 105, in init_relay                                                            
    app.mail_relay = MailRelay(config)                                                                                                                                  
  File "/home/eperez/src/git/eduid/eduid-common/src/eduid_common/api/mail_relay.py", line 48, in __init__                                                               
    self._relay = get_mail_relay(celery)                                                                                                                                
  File "/home/eperez/src/git/eduid/eduid_msg/eduid_msg/celery.py", line 37, in get_mail_relay                                                                           
    return celery_app.tasks['eduid_msg.tasks.sendmail']                                                                                                                 
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/kombu-3.0.37-py2.7.egg/kombu/utils/__init__.py", line 325, in __get__                               
    value = obj.__dict__[self.__name__] = self.__get(obj)                                                                                                               
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/app/base.py", line 663, in tasks                                     
    self.finalize(auto=True)                                                                                                                                            
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/app/base.py", line 280, in finalize                                  
    _announce_app_finalized(self)                                                                                                                                       
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/_state.py", line 73, in _announce_app_finalized                      
    callback(app)                                                                                                                                                       
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/app/base.py", line 225, in cons                                      
    return app._task_from_fun(fun, **opts)                                                                                                                              
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/app/base.py", line 270, in _task_from_fun                            
    '__wrapped__': fun}, **options))()                                                                                                                                  
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/app/task.py", line 199, in __new__                                   
    tasks.register(new(cls, name, bases, attrs))                                                                                                                        
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/app/registry.py", line 35, in register                               
    self[task.name] = inspect.isclass(task) and task() or task                                                                                                          
  File "/home/eperez/src/git/eduid/eduid-am/eduid_am/tasks.py", line 76, in __init__                                                                                    
    self.registry = PluginsRegistry(self.app.conf)                                                                                                                      
  File "/home/eperez/src/git/eduid/eduid-am/eduid_am/tasks.py", line 52, in __init__
    self.context[entry_point.name] = plugin_init(am_conf)
  File "/home/eperez/src/git/eduid/eduid-action/src/eduid_action/tou/am.py", line 65, in plugin_init
    return ToUAMPContext(am_conf['MONGO_URI'])
  File "/home/eperez/venvs/eduid-webapp/lib/python2.7/site-packages/celery-3.1.25-py2.7.egg/celery/datastructures.py", line 473, in __getitem__
    raise KeyError(key)
KeyError: 'MONGO_URI'

Comment (enriquepablo) [reply src/eduid_webapp/oidc_proofing/app.py]: This seems like another, [better solution](https://github.com/SUNET/eduid-am/compare/eperez-no-mongo?expand=1)

**Pull request merged**